packages/manager/src/features/linodes/LinodesCreate/AddonsPanel.tsx

```diff
@@ -170,9 +165,13 @@ const AddonsPanel: React.FC<CombinedProps> = (props) => {
                   }
                 />
               }
-              label="Backups"
+              label={
```

Comment:
Totally optional but I think the JSX can be moved into the same function and then we just pass it to `label`, e.g. something like

```
  const renderBackupsAddon = () => {
    const { backupsMonthly } = props;
    return (
      <Grid container spacing={2} alignItems="center">
        <Grid item>Backups</Grid>
        {backupsMonthly && (
          <Grid item>
            <Typography variant="body1">
              <Currency quantity={backupsMonthly} /> per month
            </Typography>
          </Grid>
        )}
      </Grid>
    );
  };
```

and then

```suggestion
label={renderBackupsAddon()}
```

makes it a tad cleaner